<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tetsSlots</class>
 <widget class="QWidget" name="tetsSlots">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tetsSlots</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
